core/invehicle-digital-twin/template/invehicle_digital_twin_settings.yaml

```diff
@@ -0,0 +1,11 @@
+#
+# In-Vehcile DIgital Twin Service Settings
```

Comment:
"In-Vehcile DIgital Twin Service Settings" -> "In-Vehicle Digital Twin Service Settings"

Reply:
Fixed.